www/main.js

```diff
@@ -18,8 +18,14 @@ function updateSharing() {
 
   const base64String = btoa(JSON.stringify([codeText, outputText]))
 
+  const classCode = document.querySelector('#class-code').value
+
   const url =
-    location.origin + location.pathname + 'student.html' + `#${base64String}`
+    location.origin +
+    location.pathname +
+    'student.html' +
+    `#${base64String}` +
+    `#${classCode}`
```

Comment:
This field should likely be a member of the array that is passed to `btoa` rather than appended after that field.